### DISCUSSION TITLE
Suspended Account

### DISCUSSION THREAD

**Discussion by HaroldTopre** [community/community#137148] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Account Name: haroldfromk
link to profile : https://github.com/haroldfromk/

I am writing to urgently address an issue with my account (username: haroldfromk, email: [dongik369@naver.com](mailto:dongik369@naver.com)), which has been suspended. This account is currently being used for an important ongoing project, and the suspension is causing significant disruptions.

Given the critical nature of our work, I kindly request your prompt assistance in resolving this matter. If there are any issues that need to be addressed to lift the suspension, please let me know immediately so that I can take the necessary actions.

Thank you for your understanding and swift attention to this matter. I look forward to your prompt response.

And Ticket ID: 2862435, 2866688 

I've been waiting for response since July...

**Comment by ebndev** [comment]
Hi @haroldstest,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
